backend/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
```

Comment:
The start and the main should be consistent. Depending on your purpose, either change main to server.js, or start to node app.js

However, I believe that here server.js is the main file. Thus, the file should look like this:

```json
{
  "main": "server.js",
  "scripts": {
    "start": "node server.js"
  }
}
```